docker/test/base/setup_export_logs.sh

```diff
@@ -215,7 +215,8 @@ function setup_logs_replication
 function stop_logs_replication
 {
     echo "Detach all logs replication"
-    clickhouse-client --query "select database||'.'||table from system.tables where database = 'system' and (table like '%_sender' or table like '%_watcher')" | {
+    timeout --preserve-status --signal TERM --kill-after 10m 20m \
```

Comment:
afaiu `timeout` will only be applied to `clickhouse-client`, not to `xargs` which gets stuck